src/system/SystemPacketBuffer.cpp

```diff
@@ -145,6 +145,7 @@ void PacketBufferHandle::InternalRightSize()
 
     const size_t blockSize   = usedSize + PacketBuffer::kStructureSize;
     PacketBuffer * newBuffer = reinterpret_cast<PacketBuffer *>(chip::Platform::MemoryAlloc(blockSize));
+    SYSTEM_STATS_INCREMENT(chip::System::Stats::kSystemLayer_NumPacketBufs);
```

Comment:
should this be after the null check (since I assume we use more packet buffers if we succeed to allocate only)?

Reply:
Indeed yes, done